### ISSUE TITLE
Box drawing characters

### ISSUE THREAD

**Issue by martinthomson** [martinthomson/aasvg#9]
There is some limited support for these in the tool, but that support needs a fair bit of work.  Do that work.

https://www.fileformat.info/info/unicode/block/box_drawing/list.htm

**Comment by dkg** [comment]
Just out of curiosity, is this issue about supporting box drawing characters as *input* or as some sort of rendered output?

I would personally be happy if we could use them as input, as an alternative to the current 7-bit clean `|`, `-`.

**Comment by martinthomson** [comment]
Yeah, the goal is to properly recognise all the various box drawing characters as input. Some are already recognised, but nothing comprehensive.

**Comment by dkg** [comment]
Here is a test vector, that would be useful if it could be transformed safely by aasvg:

```
╭────────────╮                           ╭────────────╮
│            │ P:Y                       │            │
│            ╞══════════════════════════▶│            │
│   Cert X   │                           │   Cert Y   │
│            │                       F:X │            │
│            │◁──────────────────────────┤            │
╰────────┬───╯                           ╰────────╥───╯
   ▲     │ F:Z                              △     ║ P:Z
   ║     │          ╭────────────╮          │     ║
   ║     │          │            │ F:Y      │     ║
   ║     └─────────▷│            ├──────────┘     ║
   ║                │   Cert Z   │                ║
   ║            P:X │            │                ║
   ╚════════════════╡            │◀═══════════════╝
                    ╰────────────╯
```

**Comment by mcr** [comment]
It looks like UTF-8 corner characters result in a blip in the line.
See: https://www.sandelman.ca/tmp/draft-richardson-rats-composite-attesters.html figure 2 as output.
Input: https://www.sandelman.ca/tmp/draft-richardson-rats-composite-attesters.txt

The ascii art was produced by asciio, "box unicode". I had no reason to do that except that it made the ASCII version slightly different.  Now that rfc7997bis says all UTF-8 is acceptable, I imagine many will want to use those characters.
